### ISSUE TITLE
Uninstall

### ISSUE THREAD

**Issue by MaryShawCh** [gitterHQ/desktop#175]
the desktop app is not uninstalling, no matter what and it keep giving notification sound.

**Comment by MadLittleMods** [comment]
> On Windows, the app has files in:
> - `C:\Users\YOUR_USER\AppData\Local\Programs\Gitter`: App install location
> - `C:\Users\YOUR_USER\AppData\Local\Gitter`: Config, see https://github.com/nwjs/nw.js/wiki/app#datapath
> 
> Unfortunately, pretty sure we leave behind files especially if you have done an update.
> 
> https://github.com/gitterHQ/gitter/issues/1366#issuecomment-240568724

**Comment by MaryShawCh** [comment]
yes, but it's not deleting, it says the program in running somewhere else.

**Comment by MadLittleMods** [comment]
@MaryShawCh You can exit the app from the icon tray in the bottom right or just end it in Task Manager.

**Comment by ChristVolo1** [comment]
Hello, is this problem resolved, or is it listed as a bug? Do we know how to remove these extra files? Again, I've uninstalled it, it's no longer showing in Programs and Features, but it still automatically opens in my toolbar with a thumbnail upon booting up, and also shows in Task Manager. I'm going to assume that since you sent me here, it's ok for me to include my screenshot: 

![uninstalled gitter but it shows in task bar and task manager](https://cloud.githubusercontent.com/assets/23203383/24535673/284415da-159b-11e7-8aff-54c179d18cb6.jpg)

**Comment by MadLittleMods** [comment]
@ChristVolo1 It's still an issue. You can clean it up fully with my instructions above, https://github.com/gitterHQ/desktop/issues/175#issuecomment-263645572
